lib/address_space/alarms_and_conditions/ConditionSnapshot.js

```diff
@@ -560,35 +555,35 @@ function prepare_date(sourceTimestamp) {
 
 function _update_sourceTimestamp(dataValue/* , indexRange*/) {
   const self = this;
-    // xx console.log("_update_sourceTimestamp = "+self.nodeId.toString().cyan+ " " + self.browseName.toString(), self.sourceTimestamp.nodeId.toString().cyan + " " + dataValue.sourceTimestamp);
+  // xx console.log(
+  //   "_update_sourceTimestamp = "+self.nodeId.toString().cyan+ " " + self.browseName.toString(),
+  //   self.sourceTimestamp.nodeId.toString().cyan + " " + dataValue.sourceTimestamp
+  // );
   self.sourceTimestamp.setValueFromSource({
     dataType: DataType.DateTime,
     value: dataValue.sourceTimestamp
   });
 }
 
 function _install_condition_variable_type(node) {
-    // from spec 1.03 : 5.3 condition variables
-    // However,  a change in their value is considered important and supposed to trigger
-    // an Event Notification. These information elements are called ConditionVariables.
+  // from spec 1.03 : 5.3 condition variables
+  // However,  a change in their value is considered important and supposed to trigger
+  // an Event Notification. These information elements are called ConditionVariables.
   node.sourceTimestamp.accessLevel = makeAccessLevel("CurrentRead");
   node.accessLevel = makeAccessLevel("CurrentRead");
 
-    // from spec 1.03 : 5.3 condition variables
-    // a condition VariableType has a sourceTimeStamp exposed property
-    // SourceTimestamp indicates the time of the last change of the Value of this ConditionVariable.
-    // It shall be the same time that would be returned from the Read Service inside the DataValue
-    // structure for the ConditionVariable Value Attribute.
+  // from spec 1.03 : 5.3 condition variables
+  // a condition VariableType has a sourceTimeStamp exposed property
+  // SourceTimestamp indicates the time of the last change of the Value of this ConditionVariable.
+  // It shall be the same time that would be returned from the Read Service inside the DataValue
+  // structure for the ConditionVariable Value Attribute.
 
   assert(node.typeDefinitionObj.browseName.toString() === "ConditionVariableType");
-  assert(node.sourceTimestamp.browseName.toString() == "SourceTimestamp");
+  assert(node.sourceTimestamp.browseName.toString() === "SourceTimestamp");
   node.on("value_changed", _update_sourceTimestamp);
 }
 
 
-import { BrowseDirection } from "lib/services/browse_service";
```

Comment:
Be carefull some import have to stay inside the install function to break cyclic dependencies